lib/metrics.js

```diff
@@ -17,9 +17,6 @@ const metricsIds = {
 
 module.exports = {
   hiddenMetrics: [
-    metricsIds.FV,
-    metricsIds.VC100,
-    metricsIds.VC85,
```

Comment:
Can we not give that freedom to the user? Let them say which metrics they want to see, both in the chart and output files?